### PR TITLE
Handle TAGS env var to build specific tags and set option to push image only

### PR DESCRIPTION
**Pull request by nghiant2710**
@Page- , @petrosagg

### CODE REVIEW

File: automation/jenkins_build.sh [hunk 1]
```diff
@@ -1,10 +1,50 @@
...
+			usage
+		}>&2
+		exit 1
+      	;;
+  esac
+done
+
+# Jenkins build steps
+cd bashbrew/
+echo $didSuccess
```
Comment (petrosagg) [inline automation/jenkins_build.sh]: This looks like it isn't used anywhere

File: automation/jenkins_build.sh [hunk 2]
```diff
@@ -1,10 +1,50 @@
...
+if [ -z "$TAGS" ]; then
+	if [ -z "$pushOnly" ]; then
+		# Build all node images
```
Comment (petrosagg) [inline automation/jenkins_build.sh]: This comment implies that it will always build node images

File: automation/jenkins_build.sh [hunk 3]
```diff
@@ -1,10 +1,50 @@
...
+		./bashbrew.sh build $LIBRARY --library=../library --namespaces=resin
+	fi
+	# Push all images
+	./bashbrew.sh push $LIBRARY --library=../library --namespaces=resin
+else
+	for tag in $TAGS; do
+		if [ ! -z "$pushOnly" ]; then
+			# Build all node images
```
Comment (petrosagg) [inline automation/jenkins_build.sh]: same here, and also it won't build *all* of them

File: automation/jenkins_build.sh (hunk 2 above)
Comment (nghiant2710) [inline automation/jenkins_build.sh]: Comments updated!

File: automation/jenkins_build.sh (hunk 1 above)
Comment (nghiant2710) [inline automation/jenkins_build.sh]: I forgot to clean it. Removed!

**Pull request merged**